features/eolearn/tests/test_features_utils.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import pytest
+
+from eolearn.features.utils import ResizeLib, ResizeMethod, spatially_resize_image
+
+
+@pytest.fixture(name="test_image", scope="module")
+def test_image_fixture():
+    """Image with a distinct value in each of the quadrants. In the bottom right quadrant there is a special layer."""
+    example = np.zeros((3, 100, 100, 4))
+    example[:, 50:, :50, :] = 1
+    example[:, :50, 50:, :] = 2
+    example[:, 50:, 50:, :] = 3
+    example[0, 50:, 50:, 0] = 10
+    example[0, 50:, 50:, 1] = 20
+    example[0, 50:, 50:, 2] = 30
+    example[0, 50:, 50:, 3] = 40
+    return example.astype(np.float32)
+
+
+@pytest.mark.parametrize("method", ResizeMethod)
+@pytest.mark.parametrize("library", ResizeLib)
+@pytest.mark.parametrize("dtype", (np.float32, np.int32, bool))
+@pytest.mark.parametrize("new_size", [(50, 50), (35, 39), (271, 271)])
+def test_spatially_resize_image_new_size(method, library, dtype, new_size):
```

Comment:
Usually we don't do this but would you mind setting types for input parameters in all test functions in this module? The problem is that PyCharm somehow doesn't understand types from decorators correctly and then marks a lot of code in these functions as "problematic".